addons/web/static/src/js/fields/basic_fields.js

```diff
@@ -1899,11 +1899,20 @@ var FieldProgressBar = AbstractField.extend({
         if (this.recordData[this.nodeOptions.current_value]) {
             this.value = this.recordData[this.nodeOptions.current_value];
         }
-        this.max_value = this.recordData[this.nodeOptions.max_value] || 100;
+        var fieldMaxValueName = this.nodeOptions.max_value;
+        var maxValueFromField = fieldMaxValueName && this.recordData[fieldMaxValueName];
+        this.max_value = maxValueFromField || 100;
+
         this.readonly = this.nodeOptions.readonly || !this.nodeOptions.editable;
-        this.edit_max_value = this.nodeOptions.edit_max_value || false;
+
+        // Boolean to toggle if we edit the numerator (value) or the denominator (max_value)
+        this.edit_max_value = this.nodeOptions.edit_max_value;
         this.title = _t(this.attrs.title || this.nodeOptions.title) || '';
-        this.edit_on_click = !this.nodeOptions.edit_max_value || false;
+
+        // Ability to edit the field through the bar
+        // Always be able to move the progress bar when in readonly mode in record
+        // and when we edit the numerator (value)
+        this.edit_on_click = this.mode === 'readonly' && !this.edit_max_value;
```

Comment:
do we really want to be able to edit this in readonly?? Also I don't see why edit_max_value should impact this.